lucet-module/src/version_info.rs

```diff
@@ -0,0 +1,92 @@
+use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use std::cmp::min;
+use std::fmt;
+use std::io;
+
+/// VersionInfo is information about a Lucet module to allow the Lucet runtime to determine if or
+/// how the module can be loaded, if so requested. The information here describes implementation
+/// details in runtime support for `lucetc`-produced modules, and nothing higher level.
+#[repr(C)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct VersionInfo {
+    major: u16,
+    minor: u16,
+    patch: u16,
+    reserved: u16,
+    version_hash: [u8; 8],
```

Comment:
Should note which portion of the hash this is

Reply:
you mean that it's the first eight ascii characters of the git hash?

Reply:
Yep